content/en/blog/2023/ambient-ebpf-redirection/index.md

```diff
@@ -0,0 +1,73 @@
+
+## How to enable eBPF redirection in Istio ambient mode
+
+Follow the [Getting Started with Ambient Mesh](/blog/2022/get-started-ambient/) instructions to set up your cluster but, when you install Istio, set the `values.cni.ambient.redirectMode` configuration parameter to `ebpf`:
+
+{{< text bash >}}
+$ istioctl install --set profile=ambient --set values.cni.ambient.redirectMode="ebpf"
+{{< /text >}}
+
+Check the `istio-cni` logs to confirm eBPF redirection is on:
+
+{{< text plain >}}
+ambient Writing ambient config: {"ztunnelReady":true,"redirectMode":"eBPF"}
+{{< /text >}}
+
+## Performance gains
+
+The latency and throughput (QPS) for eBPF mode is somewhat better than IPtables mode. The following tests were run in a kind cluster with
+a Fortio client sending requests to a Fortio server, both running in ambient mode (with debug log turned off in eBPF) and on the same k8s node.
+
+{{< image width="90%" link="./MaxQPS.png" alt="Max QPS with varying number of connections" title="Max QPS with varying number of connections" caption="Max QPS with varying number of connections" >}}
+
+The above metrics were produced with the following command:
+
+{{< text bash >}}
+$ fortio load -uniform -t 60s -qps 0 -c <num_connections> http://<fortio-svc-name>:8080
+{{< /text >}}
+
+{{< image width="90%" link="./P75-Latency-with-8000-qps.png" alt="Latency (ms) for QPS 8000 with varying number of connections" title="P75 Latency(ms) for QPS 8000 with varying number of connections" caption="P75 Latency (ms) for QPS 8000 with varying number of connections" >}}
+
+The above metrics were produced with the following command:
+
+{{< text bash >}}
+$ fortio load -uniform -t 60s -qps 8000 -c <num_connections> http://<fortio-svc-name>:8080
+{{< /text >}}
+
+## Wrapping up
+
+Both eBPF and iptables have their own advantages and disadvantages when it comes to traffic redirection. eBPF is a modern, flexible, and powerful alternative that allows for more customization in rule creation and offers better performance. However, it does require a modern kernel version (4.20 or later for redirection case) which may not be available on some systems. On the other hand, iptables is widely used and compatible with most Linux distributions, even those with older kernels. However, it lacks the flexibility and extensibility of eBPF and may have lower performance.
+
+Ultimately, the choice between eBPF and iptables for traffic redirection will depend on the specific needs and requirements of the system, as well as the user's level of expertise in using each tool. Some users may prefer the simplicity and compatibility of iptables, while others may require the flexibility and performance of eBPF.
+
+There is still plenty of work to be done, including integration with various CNI plugins, and contributions to improve the ease of use would be greatly welcomed.

```

Comment:
Close with a call to action: "Please join us in #ambient on the [Istio slack](https://slack.istio.io/)", or whatever is appropriate